### ISSUE TITLE
Israel-Palestine conflict

### ISSUE THREAD

**Issue by Floppy** [openpolitics/manifesto#426]

**Comment by Xyleneb** [comment]
Trade embargo for Israel?

**Comment by andrewdwilliams** [comment]
That depends on what we're trying to achieve with Israel-Palestine. A trade embargo would probably hit Israel pretty hard because we're no small nation, and it would royally annoy them to the point they would likely sever all diplomatic ties. If we're looking for a way to peacefully end the conflict, for instance, the two-state solution, as much as it may be a pipe dream, it would become even more difficult if we embargo trade.

**Comment by Xyleneb** [comment]
Firstly, I think there is no reason for the Israeli State to take interest in any two-state solution, discounting the economy. Secondly, I think Britain is not especially well positioned to be a mediator in the middle-east. I would invite some other country of lesser baggage and greater impartiality to do the mediating.

Do you see any other way to settle it without the use of force besides an embargo?

**Comment by Floppy** [comment]
I agree with all that; I can't remember what policy currently says (this was just moved from the todo list, and might be outdated), but I'd suggest that having something to say that we will work towards our preferred solution through the UN would be the way to go.

**Comment by philipjohn** [comment]
We currently have:

> Recognition of Palestine
> The UK should join many other countries around the world, as well as the UN, and officially recognise Palestine as a sovereign state. This is in line with our existing national preference for a two-state solution to the Israel/Palestine conflict in accordance with international law.

I would be in favour of something that would introduce penalties for a lack of progress towards a two-state solution.

**Comment by Floppy** [comment]
There may be some relevant stuff in the Vote Palestine manifesto, linked from #543
